vpc.tf

```diff
@@ -0,0 +1,6 @@
+resource "aws_vpc" "main" {
+  cidr_block = "10.0.0.0/16"
```

Comment:
why you have only used this cidr?

Reply:
as per the cidr block series we selected ip